src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -835,8 +911,11 @@ public void testCustomSimpleRootModule() throws Exception {
     public void testExecuteIgnoredModule() throws Exception {
         exit.expectSystemExitWithStatus(-2);
         exit.checkAssertionAfterwards(() -> {
+            final LocalizedMessage errorCounterMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, "Main.errorCounter",
+                    new String[] {String.valueOf(1)}, null, getClass(), null);
```

Comment:
Main.errorCounter with {1} is used about 7 times, I propose to make a field for it too, as very frequent element.